omero/sysadmins/troubleshooting.txt

```diff
@@ -331,6 +331,8 @@ more details. There are a few known possibilities:
    (:doc:`UNIX instructions <unix/install-web>` or :doc:`Windows instructions 
    <windows/install-web>`).
 
+.. _client_performance:
+
 Performance issues with the clients
```

Comment:
Maybe change to "Troubleshooting performance ..." since it makes the section and its cross-references read as though the clients have performance issues!